### PR TITLE
GCE: add EventLog registry key for docker

### PR DESCRIPTION
**Pull request by yujuhong**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
/kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by yujuhong** [comment]
@pjh please review. thanks

File: cluster/gce/windows/k8s-node-setup.psm1 [hunk 1]
```diff
@@ -1030,5 +1030,24 @@ function Verify-WorkerServices {
...
+  $reg_file = 'docker.reg'
+  Set-Content ${tmp_dir}\${reg_file} `
+'Windows Registry Editor Version 5.00
+ [HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\EventLog\Application\docker]
+"CustomSource"=dword:00000001
+"EventMessageFile"="C:\\Program Files\\docker\\dockerd.exe"
+"TypesSupported"=dword:00000007'
+
+  Log-Output "Importing registry key for Docker"
+  reg import ${tmp_dir}\${reg_file}
```
Comment (pjh) [inline cluster/gce/windows/k8s-node-setup.psm1]: I think you already tried this but just to make sure: this succeeds when done multiple times, right?

File: cluster/gce/windows/k8s-node-setup.psm1 [hunk 2]
```diff
@@ -1030,5 +1030,24 @@ function Verify-WorkerServices {
   Log_Todo "run more verification commands."
 }
 
+# Add a registry key for docker in EventLog so that log messages are mapped
+# correctly.
```
Comment (pjh) [inline cluster/gce/windows/k8s-node-setup.psm1]: Add a link to where you found this solution?

**Review by pjh — APPROVED** [review]

File: cluster/gce/windows/k8s-node-setup.psm1 (hunk 1 above)
Comment (yujuhong) [reply cluster/gce/windows/k8s-node-setup.psm1]: Yes, it always return `The operation completed successfully.`

File: cluster/gce/windows/k8s-node-setup.psm1 (hunk 2 above)
Comment (yujuhong) [reply cluster/gce/windows/k8s-node-setup.psm1]: Done

**Comment by yujuhong** [comment]
Manually applied lgtm based on @pjh's review

**Pull request merged**